README.md

```diff
@@ -100,7 +100,7 @@ Run any of the sample applications by passing to it the name that you want to gi
 ./kvsWebrtcClientMaster myChannel
 ```
 
-When the command line application prints Connection established, you can proceed to the next step.
+When the command line application prints "[KVS Master] Signaling client connection to socket established", you can proceed to the next step.
```

Comment:
Maybe removing [KVS Master] as it would be [KVS GStreamer Master] for gstreamer sample.

Reply:
Makes sense. Done.